kyverno-policy-deployment/sharing-policies.md

```diff
@@ -0,0 +1,82 @@
+# Sharing Kyverno Policies with Customers
+
+This mini-RFC documents the result of [a discussion](https://github.com/giantswarm/giantswarm/issues/26365) about Kyverno policies we deploy in customer clusters.
+
+The outcome of this RFC should be to clarify the general expectation of Kyverno policies in workload clusters, and optionally to generate additional discussions for the points that remain unclear.
+
+## TL;DR
+
+We used a Kyverno policy to implement some behavior, and didn't anticipate that customers might also use the policy. When we changed the policy for our usage, it broke the functionality for the customer.
+
+Since we will give customers lots of Kyverno policies for PSS in the future, we should let them assume that policies we deploy are intended for their usage.
```

Comment:
```suggestion
Since we will give customers lots of Kyverno policies for Pod Security Standards (PSS) in the future, we should let them assume that policies we deploy are intended for their usage.
```